files/build_templates/defaults.j2

```diff
@@ -0,0 +1,13 @@
+{# Don't use that file to generate something, it is just for holding defaults #}
```

Comment:
Why do we need a different file for setting defaults. Can't we incorporate this in init_cfg.j2?

Reply:
The default is shared between two configuration files. I don't want to explicitly incorporate debug/syslog files in the configuration. The default values here are used to fill data if the configuration in `init_cfg.j2` doesn't exist.

Reply:
I am going to remove that file. I remember we discussed that approach with Ying Xie and decided to remove defaults from that file and use defaults in-place